Sources/Extensions/UIKit/UIFont+Extensions.swift

```diff
@@ -70,10 +70,6 @@ extension UIFont {
         case normal, italic, monospace
     }
 
-    public enum Weight {
-        case ultralight, thin, light, regular, medium, semibold, bold, heavy, black
-    }
```

Comment:
Swift 4 has the exact same `Enum` with same cases. 😍

Reply:
Time to make your repo private. Apple's watching.